src/test/java/com/sfuentes/StudentTest.java

```diff
@@ -0,0 +1,96 @@
+package com.sfuentes;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StudentTest {
+
+  @Test
+  void sortByWomenAndMen_TenStudents_OnlyWomen() {
+    String[][] students = {
+        {"Daniela", "woman"},
+        {"Julia", "woman"},
+        {"Maria", "woman"},
+        {"Manuela", "woman"},
+        {"Luisa", "woman"},
+        {"Ana", "woman"},
+        {"Sonia", "woman"},
+        {"Luisa", "woman"},
+        {"Mariana", "woman"},
+        {"Sara", "woman"}
+    };
+    Student test = new Student(students);
+    Map<String, Integer> expected = new HashMap<>();
+    expected.put("Women: ", 10);
+    expected.put("Men: ", 0);
+    Map<String, Integer> actualResult = test.sortByWomenAndMen(students);
+
+    assertEquals(expected.get("Women: "), actualResult.get("Women: "));
+    assertEquals(expected.get("Men: "), actualResult.get("Men: "));
```

Comment:
instead of asserting the values on your maps you can assertEquals the whole map object

Reply:
fixed